alma-python3.9/Dockerfile

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (C) 2018-2021 CERN.
+# Copyright (C) 2022 Graz University of Technology.
+# Copyright (C) 2022 University of Münster.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+#
+ARG LINUX_VERSION=9.0
+FROM almalinux:${LINUX_VERSION}
+
+RUN dnf upgrade --refresh -y && \
+    dnf install -y \
+        dnf-plugins-core \
+        git \
+        tmux \
+        vim \
+        wget \
+        emacs-nox  \
```

Comment:
is this needed? :)

Reply:
If I remember correctly wget is needed, as far I'm concerned the other are not needed at all. I left it in, because it's there in every Dockerfile I looked in.

Reply:
It's nvm that needs curl, wget or git. As curl ist available wget is not really needed.